exporters/prometheus/src/test/resources/otel-config.yaml

```diff
@@ -1,18 +1,19 @@
 extensions:
-  health_check: {}
+  health_check:
+    endpoint: 0.0.0.0:13133
```

Comment:
how did you figure out the port? just wondering. other changes makes sense.

Reply:
That's the standard port for the health check extension: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/healthcheckextension

Additionally, in the prometheus test which sets up a collector with this config, we explicitly expose the 13133 port.